docs/dev-tools/chopsticks/overview.md

```diff
@@ -319,11 +308,9 @@ These are the methods that can be invoked and their parameters:
 
 ??? function "**dev_setHead** (hashOrNumber) — Sets the head of the blockchain to a specific hash or number"
 
-    === "Parameter"      
+    === "Parameter"
 
-        |     Name     |         Type          |               Description               |
-        | :----------: | :-------------------: | :-------------------------------------: |
-        | hashOrNumber | number \| 0x${string} | The block hash or number to set as head |
+        - `hashOrNumber` (*0x${string}*) - the block hash or number to set as head
```

Comment:
this would be `number | 0x${string}` - But i'm not the biggest fan of adding `0x` to the front of the string, because it's not a type, it's just indicating that they need to add `0x` to the hex string. So, maybe instead we can move that to the description and just do `number | string`? But I also see benefits to having it as it is. wdyt?